Project.toml

```diff
@@ -7,7 +7,7 @@ authors = [
     "Tom Jungnickel",
     "Anton Reinhard",
 ]
-version = "0.2.0"
+version = "0.2.1"
 
 [deps]
```

Comment:
Doesn't `ConstructionBase` have to be added here too?

Reply:
Now it is.

Reply:
It's still not here, I think you cherry-picked onto an outdated dev branch :see_no_evil:

Reply:
Sorry, my bad, sometimes I am too stupid for this. Thanks for taking care!